.github/workflows/pr-checks.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
       fail-fast: false
       matrix:
         module: ${{ fromJson(needs.module-diff.outputs.modules) }}
-        go: [ "~1.24.11" ]
+        go: [ "1.24" ]
```

Comment:
might make sense to just remove it from the matrix and use directly from go.work, but as you wish.